prowler/lib/check/custom_checks_metadata.py

```diff
@@ -18,9 +18,63 @@
                         "Severity": {
                             "type": "string",
                             "enum": valid_severities,
-                        }
+                        },
+                        "CheckTitle": {
+                            "type": "string",
+                        },
+                        "Description": {
+                            "type": "string",
+                        },
+                        "Risk": {
+                            "type": "string",
+                        },
+                        "RelatedUrl": {
+                            "type": "string",
+                        },
+                        "Remediation": {
+                            "type": "object",
+                            "properties": {
+                                "Code": {
+                                    "type": "object",
+                                    "properties": {
+                                        "CLI": {
+                                            "type": "string",
+                                        },
+                                        "NativeIaC": {
+                                            "type": "string",
+                                        },
+                                        "Other": {
+                                            "type": "string",
+                                        },
+                                        "Terraform": {
+                                            "type": "string",
+                                        },
+                                    },
+                                    "required": [
+                                        "CLI",
+                                        "NativeIaC",
+                                        "Other",
+                                        "Terraform",
+                                    ],
+                                },
+                                "Recommendation": {
+                                    "type": "object",
+                                    "properties": {
+                                        "Text": {
+                                            "type": "string",
+                                        },
+                                        "Url": {
+                                            "type": "string",
+                                        },
+                                    },
+                                    "required": ["Text", "Url"],
```

Comment:
Are we sure of this? I think we want just to override the fields present in the custom metadata file.

Reply:
You are true, I think no field should be required.